server/api/room.js

```diff
@@ -16,10 +17,60 @@ router.get('/', async (req, res, next) => {
 // GET - /api/room/create/
 // Makes a new room and returns it
 // Maybe change to POST if we want the firebase room name on the room
-router.post('/create/:roomId', async (req, res, next) => {
+// router.post('/create/:roomId', async (req, res, next) => {
```

Comment:
Not sure if this commented out code is necessary anymore? if you know it isn't we can take it out but not a big deal

Reply:
it's not needed anymore! i deleted it.